### PR TITLE
[FIX] stock_dropshipping: create dropship rules

### PR DESCRIPTION
**Pull request by remybar**
Same issue than for commit 203bb4d9c11 but for dropship rules instead of
missing dropship types.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock_dropshipping/models/res_company.py [hunk 1]
```diff
@@ -86,8 +86,9 @@ def _create_dropship_rule(self):
         dropship_vals = []
         for company in self:
             dropship_picking_type = self.env['stock.picking.type'].search([
-                ('name', '=', 'Dropship'),
-                ('company_id', '=', company.id),
+                ("company_id", "=", company.id),
```
Comment (amoyaux) [inline addons/stock_dropshipping/models/res_company.py:89]: simple quote ''

Comment (remybar) [reply addons/stock_dropshipping/models/res_company.py]: thanks, done!

**Comment by amoyaux** [comment]
robodoo r+

**Comment by avg-odoo** [comment]
Urgent [OPW-2356935](https://www.odoo.com/web#active_id=2356935&cids=1%2C2%2C3&id=2356935&model=project.task&menu_id=) related to this PR 
Thanks in advance @Whenrow

**Comment by amoyaux** [comment]
robodoo retry

**Pull request closed without merging**